charts/turing/README.md

```diff
@@ -0,0 +1,73 @@
+# turing
```

Comment:
Shall we parameterise this file with `README.md.gotmpl` and `helm-docs`?

Reply:
We don't have this for any of the CaraML components. 🤔 @mbruner any thoughts on this? Shall we add it in?

Reply:
@krithika369 we have it for a few charts `routes`, `common`

Reply:
Added in the template for Merlin and Turing.